### ISSUE TITLE
How to open embedded link in ZefyrView?

### ISSUE THREAD

**Issue by badaween-ahmed** [memspace/zefyr#257]
I couldn't find a way to launch the url from an embedded link? There is no onTap callback or something similar in `ZefyrView()`. It's also not in the example app.

**Comment by CanRau** [comment]
Is that what you're requesting https://github.com/memspace/zefyr/pull/211 ?

**Comment by badaween-ahmed** [comment]
@CanRau  Yes. And I saw that PR but I couldn't figure it out how to implement. I wanna be able to open the url (using url launhcer) when I tap on the embedded link.

**Comment by CanRau** [comment]
Sorry I'm pretty new to Flutter and haven't tried it so can't help any further at the moment

**Comment by badaween-ahmed** [comment]
@CanRau I think it hasn't been implemented. Please give my question thumbs up so it gets the attention.

**Comment by raj457036** [comment]
https://github.com/memspace/zefyr/issues/41#issuecomment-600467646

Check my solution at above link

**Comment by badaween-ahmed** [comment]
That's possible in `ZefyrEditor()` where you can get controller. But there is no controller in `ZefyrView()`.

**Comment by raj457036** [comment]
you can set the mode to ZefyrMode.select or view
